_Appendix/glossary.md

```diff
@@ -598,9 +586,6 @@ NCHC
 NIST
 : National Institute of Standards and Technology
 
-NISTIR
-: National Institute of Standards and Technology Interagency Report
```

Comment:
We do use NISTIR to reference NIST work.  We use, for example,  NIST [IR 7863].  Probably, we should use [NISTIR 7863] instead of "IR" because 'IR'  is define as "infra red".

Reply:
If we're keeping it, I think the official name is "NIST Internal or Interagency Report"

Reply:
Good catch! That's why it didn't show up in my search. We should keep it.

We had previously discussed it being "NIST [IR XXXX]" because the document series is "IR".